data/editor.config

```diff
@@ -579,6 +579,10 @@
       <type id="leisure-garden" can_add="no">
         <include group="poi_internet" />
       </type>
+      <!-- Not addable as mapping as a node is uncommon (use one of the hotels type) -->
+      <type id="leisure-resort" can_add="no">
```

Comment:
27% are mapped as nodes currently.
IMO its better if a user will add a `resort` node e.g. at its entrance or somewhere inside rather than they will do the same but with a `tourism=hotel` tagging...

Reply:
But a hotel is a node within a resort area, no?

Reply:
Ok, probably can allow here